schema/data-auth.json

```diff
@@ -78,4 +78,7 @@
     ["Stop Broadcast","AUTH_STOPBROADCAST", true],
     ["Add Quotes","AUTH_ADD_QUOTES", true],
     ["Upload Music - Manual","AUTH_UPLOADMUSICMANUAL", true]
+    ["User can tracklist own/current show","AUTH_TRACKLIST_OWN", true],
+    ["User can tracklist for any show/timeslot","AUTH_TRACKLIST_ALL", true]
```

Comment:
Is it worth making SIS use these permissions as well? Otherwise it's a bit inconsistent

Reply:
Hmm, there's an overall need to consolidate all of the varying different functions that do the same all over the shop, but I want to say that's out of scope for this PR.

Reply:
Fair, file that as a future issue

Reply:
#861